man/systemd.netdev.xml

```diff
@@ -1979,6 +1986,86 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[Batadv] Section Options</title>
+    <para>The [Batadv] section only applies for
+    netdevs of kind <literal>batadv</literal> and accepts the
+    following keys:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>GatewayMode=</varname></term>
+        <listitem>
+          <para>The GatewayMode is a string of either <option>off</option>, <option>server</option> or
+          <option>client</option>.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>Aggregation=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. Enables or disables aggregation of OGMs. Defaults to true.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>BridgeLoopAvoidance=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. Enables or disables avoidance loops on bridges. Defaults to true.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>DistributedArpTable=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. Enables or disables the distributed arp table. Defaults to true</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>Fragmentation=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. Enables or disables fragmentation. Defaults to true</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>HopPenalty=</varname></term>
+        <listitem>
+          <para>The hop penalty setting allows to modify <command>batctl</command> preference for multihop
+          routes vs. short routes. The value is applied to the TQ of each forwarded OGM, thereby
+          propagating the cost of an extra hop (the packet has to be received and retransmitted
+          which costs airtime). A higher hop penalty will make it more unlikely that other nodes
+          will choose this node as intermediate hop towards any given destination.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>OrigInterval=</varname></term>
```

Comment:
we generally try to unify on using usec granularity internally for all time values. And by default expose seconds to the outside, if the user doesn't specify an explicit time unit. And we try to suffic the fields with their default unit. i.e. please call this OriginalIntervalSec= and parse this with config_parse_sec() and store as usec_t internally.

we value internal consistency quite a lot, hence please try to stick to using the same time unit everywhere, except where there are strong reasons not to.